f/frizzle/comapeo/README.md

```diff
@@ -0,0 +1,46 @@
+# CoMapeo
```

Comment:
Out of curiosity, does the Windmill UI surface this README anywhere?

If not, i wonder what affordance Windmill provides for script-level info that it will surface.

Reply:
It doesn't seem to. I don't see it in the CLI diff nor do I find it anywhere in the UI after syncing.

I did find that `description` in `.script.yaml` is surfaced on the script page. But it's just a string; linebreaks and markdown don't seem to work.